### ISSUE TITLE
breakpoint is not respected in polyglot app

### ISSUE THREAD

**Issue by miloslavskacel** [oracle/graal#2574]
**Describe the issue**

App (graalvm-demos\functionGraphDemo)  execution does not stop on a breakpoint set in Chrome debugger.

**Steps to reproduce the issue**

```
git clone https://github.com/graalvm/graalvm-demos.git
cd graalvm-demos/functionGraphDemo
npm i
```

Edit `server.js` to comment out lines 6,7,9,10 and to add console.log into `app.post('/graph'`  handler.
It should look like:
```
var express = require('express');
var bodyParser = require("body-parser");
var app = express();
var APP_PORT = 8088;

// Polyglot.evalFile("ruby", "validator.rb");
// var Validator = Polyglot.import('Validator');

// Polyglot.evalFile("R", "functionGraph.r");
// var plotFunction = Polyglot.import('plotFunction');

app.use(bodyParser.urlencoded({ extended: false }));
app.use(bodyParser.json());

app.post('/graph', function (req, res) {
    console.log("I processing POST graph", req.body);
    res.send("Ruby and R not yet ready in GraalVM 20.1");
    // let expr = req.body.expr;
... abbreviated
```

Execute:
```
node --jvm --polyglot  --inspect server.js
Debugger listening on port 9229.
To start debugging, open the following URL in Chrome:
    chrome-devtools://devtools/bundled/js_app.html?ws=127.0.0.1:9229/N47JTonvwnXXiZqnrrFmQCLclHdAkV9LIDr2DFCkA_s
```
Open URL for debugging (note to use just devtools:// as URL schema, #2573)
```
devtools://devtools/bundled/js_app.html?ws=127.0.0.1:9229/N47JTonvwnXXiZqnrrFmQCLclHdAkV9LIDr2DFCkA_s
```
Debugger is opened.
Add directory `functionGraphDemo` to the `Filesystem` panel by `+` button.
Set the breakpoints as on screenshot:

![debugger-1](https://user-images.githubusercontent.com/2547672/84907238-4024a780-b0b3-11ea-820c-6d55b402dea6.png)

Open the app URL in Chrome: `http://localhost:8088/`
Press `Draw Function`.
Execution shall be stopped at the breakpoint, but it is not.

**Describe GraalVM and your environment:**

```
node --version
v12.15.0

which node
C:\java\graalvm-ce-java11-20.1.0\bin\node.CMD

java --version
openjdk 11.0.7 2020-04-14
OpenJDK Runtime Environment GraalVM CE 20.1.0 (build 11.0.7+10-jvmci-20.1-b02)
OpenJDK 64-Bit Server VM GraalVM CE 20.1.0 (build 11.0.7+10-jvmci-20.1-b02, mixed mode, sharing)
```

**More details**
Use  `inspect-brk`.

```
node --jvm --polyglot  --inspect-brk server.js
```
Now `file://C:\workspace\prototypes\graalvm-demos\functionGraphDemo\server.js` is opened by Chrome.
Set the breakpoint into `app.post('/graph'`  handler.

![debugger-2](https://user-images.githubusercontent.com/2547672/84908672-0a80be00-b0b5-11ea-837c-3f5e20eeafc1.png)

Open the app URL in Chrome: `http://localhost:8088/`
Press `Draw Function`.
Execution is now stopped at the breakpoint.

**Comment by boris-spas** [comment]
Tracking internally as Issue GR-24942.

**Comment by entlicher** [comment]
I've successfully reproduced the bug on Windows. It looks like file path issue.
The reason why it works with `--inspect-brk` is, that it suspends at the beginning of the server.js file and opens it with the `file://C:\...` path. You submit breakpoint into that file then where the breakpoints work. Breakpoints do not work in the server.js file opened from the Filesystem.

**Comment by entlicher** [comment]
Fortunately, this issue is already resolved in 20.2-dev.
